forespoersel-besvart/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/forespoerselbesvart/ForespoerselBesvartFraSimbaRiver.kt

```diff
@@ -0,0 +1,61 @@
+package no.nav.helsearbeidsgiver.inntektsmelding.forespoerselbesvart
+
+import kotlinx.serialization.json.JsonElement
+import no.nav.helsearbeidsgiver.felles.EventName
+import no.nav.helsearbeidsgiver.felles.Key
+import no.nav.helsearbeidsgiver.felles.json.krev
+import no.nav.helsearbeidsgiver.felles.json.les
+import no.nav.helsearbeidsgiver.felles.json.toPretty
+import no.nav.helsearbeidsgiver.felles.metrics.Metrics
+import no.nav.helsearbeidsgiver.felles.rapidsrivers.river.ObjectRiver
+import no.nav.helsearbeidsgiver.felles.utils.Log
+import no.nav.helsearbeidsgiver.utils.json.serializer.UuidSerializer
+import no.nav.helsearbeidsgiver.utils.log.sikkerLogger
+import java.util.UUID
+
+data class BesvartSimbaMelding(
+    val eventName: EventName,
+    val transaksjonId: UUID,
+    val forespoerselId: UUID,
+)
+
+class ForespoerselBesvartFraSimbaRiver : ObjectRiver<BesvartSimbaMelding>() {
+    private val sikkerLogger = sikkerLogger()
+
+    override fun les(json: Map<Key, JsonElement>): BesvartSimbaMelding? =
+        if (setOf(Key.BEHOV, Key.DATA, Key.FAIL).any(json::containsKey)) {
+            null
+        } else {
+            BesvartSimbaMelding(
+                eventName = Key.EVENT_NAME.krev(EventName.INNTEKTSMELDING_MOTTATT, EventName.serializer(), json),
+                transaksjonId = Key.UUID.les(UuidSerializer, json),
+                forespoerselId = Key.FORESPOERSEL_ID.les(UuidSerializer, json),
+            )
+        }
+
+    override fun BesvartSimbaMelding.haandter(json: Map<Key, JsonElement>): Map<Key, JsonElement>? {
+        Metrics.forespoerslerBesvartFraSimba.inc()
+
+        return notifikasjonHentIdMelding(transaksjonId, forespoerselId)
+    }
+
+    override fun BesvartSimbaMelding.haandterFeil(
+        json: Map<Key, JsonElement>,
+        error: Throwable,
+    ): Map<Key, JsonElement>? {
+        sikkerLogger.error(
+            "Klarte ikke umiddelbart markere forespørsel som besvart. " +
+                "Event fra helsebro skal løse dette. Aktuell melding:\n${json.toPretty()}",
+        )
```

Comment:
Denne logger bare til sikker logger, mens de andre Riverne typisk logger feil både til vanlig og sikker. Ser det har vært på denne måten her i den forrige løseren også, men tenkte på at det kanskje kunne være nyttig i feilsøkingsøyemed med en sånn `Se sikker logg for mer info.`-melding i vanlig logg sånn som i `ForespoerselBesvartFraSpleisRiver`?

Reply:
Jeg kan legge det til. Vi har ikke alltid vært like flinke å logge til begge.